ci-operator/config/openshift/release/openshift-release-master__ocp-4.5.yaml

```diff
@@ -1,9 +1,29 @@
 base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.5"
+    namespace: ocp
+    tag: base
   dev-scripts:
     cluster: https://api.ci.openshift.org
     name: test
     namespace: openshift-kni
     tag: dev-scripts
+  installer:
```

Comment:
These will be pulled in by the `tag_specification` -- does something break if you don't explicitly list them here?

Reply:
@stevekuznetsov I didn't know that worked.  I will update and check

Reply:
hmm
```
11m         16m          17      e2e-vsphere-ipi-conf-vsphere-vips.1610c8499e57c315   Pod    spec.containers{test}                    Warning   Failed         kubelet, origin-ci-ig-n-fc9g   Error: ImagePullBackOff
2m          16m          56      e2e-vsphere-ipi-conf-vsphere-vips.1610c8499e576b95   Pod    spec.containers{test}                    Normal    BackOff        kubelet, origin-ci-ig-n-fc9g   Back-off pulling image "docker-registry.default.svc:5000/ci-op-y7mfr5h7/stable@sha256:e984a8e70231118a960def8b999177bfa3060d82cf1486b730bac144a04f0e12"
```

Reply:
@stevekuznetsov I am missing something?

15m         16m          3       e2e-vsphere-ipi-conf-vsphere-vips.1610c8462fe266e9   Pod    spec.containers{test}                    Warning   Failed         kubelet, origin-ci-ig-n-fc9g   Failed to pull image "docker-registry.default.svc:5000/ci-op-y7mfr5h7/stable@sha256:e984a8e70231118a960def8b999177bfa3060d82cf1486b730bac144a04f0e12": rpc error: code = Unknown desc = unauthorized: authentication required

Reply:
@stevekuznetsov I added the images back and at least there are no auth failures.  I don't really expect vSphere IPI to pass all the tests atm